### DISCUSSION TITLE
InkStitch installation o Mac with inkstitch-v2.2.0-capitan-catalina-osx

### DISCUSSION THREAD

**Discussion by Flintstone74** [inkstitch/inkstitch#2205] [Q&A]
Help I have installed the file "inkstitch-v2.2.0-capitan-catalina-osx" but I cant find where on the HD it has been installed. It has not created a dir and did not give me an option to name a dir or install at the location the original file was stored?

I need to move these files to the Inkscape ext dir ?

Any help appreciated OS is High Sierra 10.13.6

Flint

**Comment by kaalleen** [comment]
Ink/Stitch is a Inkscape plugin. So it should be in the Inkscape extensions menu after the installation.
